lib/parser/current.rb

```diff
@@ -9,24 +9,6 @@ def warn_syntax_deviation(feature, version)
   end
 
   case RUBY_VERSION
-  when /^1\.8\./
```

Comment:
This should be kept and raise an error.

Reply:
Wouldn't it be better to put something in `lib/parser.rb` which either raises an error or prints a warning if run on 1.8/1.9?

If we put it in `lib/parser/current.rb`, the error will only be seen if that file is loaded.

Reply:
Sure, that's better.